src/app/admin/usersTable/UsersTable.tsx

```diff
@@ -230,6 +231,26 @@ const UsersTable: React.FC = () => {
         void fetchAndDisplayUserData();
     }, [fetchAndDisplayUserData]);
 
+    useEffect(() => {
+        void fetchAndDisplayUserData();
+        // This requires that the profiles tables have Realtime turned on
```

Comment:
I don't think this comment is needed anymore as we manage realtime through migrations